src/test/cypress/support/pageobjects/exercises/text/TextExerciseCreationPage.ts

```diff
@@ -0,0 +1,66 @@
+import { BASE_API } from './../../../constants';
+import { Dayjs } from 'dayjs';
+import { POST } from '../../../constants';
+import { dayjsToString } from '../../../utils';
+/**
+ * A class which encapsulates UI selectors and actions for the text exercise creation page.
+ */
+export class TextExerciseCreationPage {
+    /**
+     * @param title the title of the text exercise
+     */
+    typeTitle(title: string) {
+        cy.get('#field_title').clear().type(title);
+    }
+
+    setReleaseDate(date: Dayjs) {
+        this.typeDate(date, 'releaseDate');
+    }
+
+    setDueDate(date: Dayjs) {
+        this.typeDate(date, 'dueDate');
+    }
+
+    setAssessmentDueDate(date: Dayjs) {
+        this.typeDate(date, 'assessmentDueDate');
+    }
+
+    typeMaxPoints(maxPoints: number) {
+        cy.get('#field_points').type(maxPoints.toString());
+    }
+
+    checkAutomaticAssessmentSuggestions() {
+        cy.get('#automatic_assessment_enabled').check();
+    }
+
+    typeProblemStatement(statement: string) {
+        this.typeText('#problemStatement', statement);
+    }
+
+    typeExampleSolution(statement: string) {
+        this.typeText('#sampleSolution', statement);
+    }
+
+    typeAssessmentInstructions(statement: string) {
+        this.typeText('#gradingInstructions', statement);
+    }
+
+    create() {
+        cy.get('[jhitranslate="artemisApp.textExercise.exampleSubmissionsRequireExercise"]').should('be.visible');
```

Comment:
I think asserting this is visible should be done in the test not the pageobject.

Reply:
Makes sense